libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/TableFileStats.java

```diff
@@ -16,7 +16,7 @@ public class TableFileStats implements DataSource<FileStat> {
   @Override
   public Dataset<FileStat> get() {
     return spark
-        .sql(String.format("SELECT file_path, file_size_in_bytes FROM %s.files", tableName))
+        .sql(String.format("SELECT file_path, file_size_in_bytes FROM %s.data_files", tableName))
```

Comment:
I would imagine this code layout to be either

```
libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/TableFileDataSourceImpl.java
libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/TablePartitionDataSourceImpl.java
libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/dao/FileStat.java
libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/dao/PartitionStats.java
```
given the last two are part of your "spec". If there are just in-memory DTOs then replace `dao` with `dto`